admin-dev/themes/new-theme/js/pages/product/index.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+// This had to be commented because of TS2451: Cannot redeclare block-scoped variable '$'.
+// But in other index.ts there is no such issue
+// const {$} = window;
```

Comment:
ping @NeOMakinG really, really weird

Reply:
After investigate through the issue, it seems that tsconfig.json is loaded all files located in subdirectories where the tsconfig.json file is defined.
It ignores the webpack configuration.
FYI: https://github.com/TypeStrong/ts-loader#onlycompilebundledfiles

Reply:
This is an error from TS, it should interpret every bundles separately but I've another problem with this PR, going to review it rn

Reply:
But how come in some other index.ts files it works fine then? This one seems even simpler since it doesn't import external components Or could it be that it is bundled together with the other js code in the `product` folder?

Maybe if we rename it `listing.ts` there will be less trouble? Or should it be fixed in the TS config?